packages/@sanity/core/src/commands/dataset/copyDatasetCommand.js

```diff
@@ -164,13 +164,9 @@ export default {
       output.print(
         `Copying dataset ${chalk.green(sourceDatasetName)} to ${chalk.green(targetDatasetName)}...`
       )
+      output.print(`Job ${chalk.green(response.jobId)} started`)
 
       if (flags.detach) {
-        output.print('Copy initiated.')
-        output.print(
-          `\nRun:\n\n    sanity dataset copy --attach ${response.jobId}\n\nto watch attach`
```

Comment:
I thought this was a good instruction to have. It gave the user a call to action.

Reply:
I opted to remove it since we are now printing the job id on a separate line and we have good detailing of the command in the help screen. As such, it becomes a bit verbose and presents an awkward symmetry with the "attached" copy.